### DISCUSSION TITLE
How to hide the underline of a file or folder

### DISCUSSION THREAD

**Discussion by huangcs427** [ohmyzsh/ohmyzsh#10545] [Help] [answered]
As shown in the title，When I zoom in on my terminal the underscores of files or folders fill up the screen, which is very unsightly。
This seems to be a problem with windows terminal, because I did not reproduce in tabby.

As shown in the figure below：
![image](https://user-images.githubusercontent.com/31842636/147740975-6fa03763-9c77-49d0-9803-dae46149d98b.png)

My Terminal: Windows Terminal.

Thank you very much for your answer

**Comment by mcornella** [comment, marked as answer]
That styling is done by `zsh-syntax-highlighting`, in particular by the [`main` highlighter](https://github.com/zsh-users/zsh-syntax-highlighting/tree/master/highlighters/main#readme). What you want is change the `path` style, which is [`underline` by default](https://github.com/zsh-users/zsh-syntax-highlighting/blob/c7caf57ca805abd54f11f756fda6395dd4187f8a/highlighters/main/main-highlighter.zsh#L40). There's an example change in the linked README:

```sh
# Declare the variable
typeset -A ZSH_HIGHLIGHT_STYLES

# To have paths colored instead of underlined
ZSH_HIGHLIGHT_STYLES[path]='fg=cyan'
```

You'd put this in your zshrc file. To change the style, you can use `fg=color` for the foreground color, `bg=color` for the background color, `bold`, `underline`, or `standout` (it inverts the foreground and background color on most terminals). You can add multiple keywords with a comma: `fg=red,bold`.

[More info about specific formatting here](https://zsh.sourceforge.io/Doc/Release/Zsh-Line-Editor.html#Character-Highlighting).

**Reply by huangcs427** [reply]
Thank you, This solves my problem
